config/locales/client.en.yml

```diff
@@ -1486,12 +1483,49 @@ en:
           error: "There was an error generating Invite link"
           max_redemptions_allowed_label: "How many people are allowed to register using this link?"
           expires_at: "When will this invite link expire?"
+
+        invite:
+          new_title: "Create Invite"
+          edit_title: "Edit Invite"
+          show_link: "Invite Link"
+
+          instructions: "Share this link to instantly grant access to this site:"
+          no_invite_link: "The invite link will show up here after created"
+          copy_link: "copy link"
+          expires_at_time: "Your invite link expires in %{time}."
+
+          show_advanced: "▶ Show Advanced Options"
```

Comment:
I recommend you move the icons out of the translation key and use a font awesome helper.